### PR TITLE
Graph clustering

### PR DESCRIPTION
**Pull request by Manon3005**
https://github.com/VCityTeam/UD-Viz/issues/824
https://github.com/VCityTeam/UD-Viz/issues/825
https://github.com/VCityTeam/UD-Viz/issues/826
https://github.com/VCityTeam/UD-Viz/issues/827
https://github.com/VCityTeam/UD-Viz/issues/828
https://github.com/VCityTeam/UD-Viz/issues/829
https://github.com/VCityTeam/UD-Viz/issues/830
https://github.com/VCityTeam/UD-Viz/issues/831

### CODE REVIEW

**Comment by DiegoVinasco** [comment]
@Manon3005 would also like to add #835 to this PR if possible

File: examples/assets/config/layer/3DTiles_point_cloud.json
Comment (DiegoVinasco) [inline examples/assets/config/layer/3DTiles_point_cloud.json:1]: I don't even understand how we were able to parse this file before with a syntax error like that

**Review by DiegoVinasco — CHANGES_REQUESTED** [review]
The PR is in good shape. There's just a few things that need to be modified before the PR is accepted, please see my comments.

Once those are done we should follow the [PR submission checklist](https://github.com/VCityTeam/UD-Viz/blob/master/docs/static/Contributing.md#submiting-pull-request-pr) to make sure everything is ready. Then we can merge the PR!

- [x] Comments
- [ ] [Checklist](https://github.com/VCityTeam/UD-Viz/blob/master/docs/static/Contributing.md#submiting-pull-request-pr)
- [ ] PR/MR

File: packages/widget_sparql/src/view/D3GraphCanvas.js [hunk 1]
```diff
@@ -303,94 +1496,179 @@ export class D3GraphCanvas extends THREE.EventDispatcher {
...
+    graph.nodeCircle
+      .attr('cx', function (d) {
+        return d.x;
+      })
+      .attr('cy', function (d) {
+        return d.y;
+      });
+
+    graph.link.attr('d', function (d) {
+      const getNormalVec = (A, B) => {
```
[the remark this answers is not in the document]
Comment (Manon3005) [reply packages/widget_sparql/src/view/D3GraphCanvas.js:1513]: Docstring added!

File: examples/assets/config/layer/3DTiles_temporal.json
[the remark this answers is not in the document]
Comment (Manon3005) [reply examples/assets/config/layer/3DTiles_temporal.json:1]: Changes made!

**Review by DiegoVinasco — APPROVED** [review]

**Pull request merged**